ui/src/components/SampleView/ApertureInput.js

```diff
@@ -25,7 +25,6 @@ export default class ApertureInput extends React.Component {
         <Form.Select
           className={inputCSS}
           style={{ float: 'none' }}
-          id="puckFilter"
           defaultValue={this.props.aperture}
```

Comment:
It seems it should be `value={this.props.aperture}` instead of `defaultValue={this.props.aperture}`. Otherwise it doesn’t get updated in ui when aperture is changed from diffractometer.

Reply:
Correct, thanks for having a look !

Reply:
@meghdadyazdi Have you manage to run this PR yet ?

Reply:
Not yet actually. As I was working on aperture hardwareObject I noticed this issue.